### PR TITLE
refactor(ui): move global css files & introduce default layout component

### PR DESCRIPTION
**Pull request by henil0604**

### CODE REVIEW

File: src/lib/styles/theme.css
Comment (ardentkilnfire) [inline src/lib/styles/theme.css:1]: Suggestion: Prefix custom properties with `clr` or `bg`.
Example: `--clr-primary` or `--bg-card`

Comment (henil0604) [reply src/lib/styles/theme.css:1]: +1. i think we can do this without affecting class names in shadcn components.

Comment (henil0604) [reply src/lib/styles/theme.css:1]: d10152f should resolve this

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -5,6 +5,8 @@
 	const { children } = $props();
 </script>
 
-<DefaultLayout {children} />
```
Comment (henil0604) [inline src/routes/+layout.svelte]: this somehow was giving a `warning (with red text)` about not using `slot` or `@render` in the console. but somehow was rendering perfectly.

File: tailwind.config.ts [hunk 2]
```diff
@@ -15,44 +15,44 @@ const config: Config = {
...
-					foreground: 'hsl(var(--card-foreground) / <alpha-value>)'
+					DEFAULT: 'hsl(var(--clr-card) / <alpha-value>)',
+					foreground: 'hsl(var(--clr-card-foreground) / <alpha-value>)'
 				}
 			},
 			borderRadius: {
-				lg: 'var(--radius)',
-				md: 'calc(var(--radius) - 2px)',
-				sm: 'calc(var(--radius) - 4px)'
+				lg: 'var(--size-radius)',
```
Comment (ardentkilnfire) [inline tailwind.config.ts:53]: `--br-radius` is more suitable :thinking: 
- `br` as a short-form for `border`

Comment (henil0604) [reply tailwind.config.ts:53]: +1

Comment (henil0604) [reply tailwind.config.ts]: f96b675 should resolve this

File: src/routes/+layout.svelte (hunk 1 above)
Comment (ardentkilnfire) [reply src/routes/+layout.svelte]: :man_shrugging:

**Comment by ardentkilnfire** [comment]
Everything seems good now :smile:

**Review by ardentkilnfire — APPROVED** [review]

**Pull request merged**